paperqa/litqa.py

```diff
@@ -43,13 +43,13 @@ def make_mc_options(
             split_distractors = [d.strip("'[ ]\"") for d in distractors.split(",")]
         options: list[str] = split_distractors
     else:
-        options = distractors
+        # We are going to modify options in-place, so copy it first
+        options = distractors.copy()
 
     if ideal == "null":
         if not unsure_option:
             raise ValueError(
-                'Dataset configured for "unsure" options via '
-                'ideal="null", please specify "unsure_option".'
+                'Dataset configured for "unsure" options via ideal="null", please specify "unsure_option".'
```

Comment:
Can you undo these string wrapping changes (and messages below)? It no longer fits in 88 chars with these